extensions/src/test/java/org/pentaho/platform/web/http/filters/PentahoAwareCharacterEncodingFilterTest.java

```diff
@@ -68,7 +67,7 @@ public void doFilter_ignoreFlagIsSet() throws Exception {
     assertEquals( NEW_ENCODING, request.getCharacterEncoding() );
   }
 
-  @Ignore
+
```

Comment:
Remove these extra spaces when you remove @ignore annotation
Line number 59 and 70